### DISCUSSION TITLE
Not able to get YOLO Badge

### DISCUSSION THREAD

**Discussion by NayraRagan** [drknzz/GitHub-Achievements#231] [Q&A] [answered]
NOt able to get yolo badge even after doing what the badge asks

**Comment by NagarAryan** [comment, marked as answer]
I thinks so go for merging for a pull request with the reviewer accepting the solution I guess that would be very helpful

**Comment by bettyc0der** [comment]
I starred your repo please star mine
https://github.com/salcedocharlotte/CIS261-Course-Project
